.github/workflows/scripts/css_check.py

```
@@ -7,124 +7,252 @@
+        "line_number",
+        "violation_type",
+        "code_snippet",
+        "description",
+    ],
 )
+CSSCheckResult = namedtuple("CSSCheckResult", ["violations"])
 
 
-def check_embedded_css(content: str) -> list:
-    """Check for embedded CSS in the content.
+def strip_comments(line: str, in_block_comment: bool) -> tuple[str, bool]:
```

Ruff (0.14.8)

Boolean-typed positional argument in function definition

(FBT001)